instance/tests/api/test_openedx_appserver.py

```diff
@@ -145,7 +145,7 @@ def test_get_details(self):
         self.assertIn(('public_ip', None), server_data)
         # The API call will try to start the server, which will fail, since we're
         # not actually talking to an OpenStack instance when unit tests are running
-        self.assertIn(('status', 'failed'), server_data)
+        self.assertIn(('status', 'unknown'), server_data)
```

Comment:
Actually, it looks like these tests were failing on `master` and this is a drive-by fix. Is that right? If so, I think we need to actually determine the cause instead of just changing the test. I think that the test code was correct as written, and that `failed` is more useful than `unknown`. Same down below for the 404 error.